.github/scripts/nm-run-lm-eval-gsm-hf-baseline

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# We can use this script to compute baseline accuracy on GSM for transformers.
+#
+# Make sure you have lm-eval-harness installed:
+#   pip install git+https://github.com/EleutherAI/lm-evaluation-harness.git@9516087b81a61d0e220b22cc1b75be76de23bc10
+
+usage() {
+    echo``
+    echo "Runs lm eval harness on GSM8k using huggingface transformers."
+    echo "This pathway is intended to be used to create baselines for "
+    echo "our automated nm-test-accuracy workflow"
+    echo
+    echo "usage: ${0} <options>"
+    echo
+    echo "  -m    - huggingface stub or local directory of the model"
+    echo "  -b    - batch size to run the evaluation at"
+    echo "  -d    - device to use (e.g. cuda, cuda:0, auto, cpu)"
+    echo "  -l    - limit number of samples to run"
+    echo "  -f    - number of fewshot samples to use"
+    echo
+}
+
+while getopts "m:b:d:l:f:" OPT; do
+  case ${OPT} in
+    m ) 
+        MODEL="$OPTARG"
+        ;;
+    b ) 
+        BATCH_SIZE="$OPTARG"
+        ;;
+    d ) 
+        DEVICE="$OPTARG"
+        ;;
+    l ) 
+        LIMIT="$OPTARG"
+        ;;
+    f ) 
+        FEWSHOT="$OPTARG"
+        ;;
+    \? ) 
+        usage
+        exit 1
+        ;;
+  esac
+done
+
+lm_eval --model hf \
+  --model_args pretrained=$MODEL \
+  --tasks gsm8k --num_fewshot $FEWSHOT --limit $LIMIT \
```

Comment:
`lm-eval` is expecting (looking at [source code](https://github.com/EleutherAI/lm-evaluation-harness/blob/305fb6367319cb9279bf4820b0c0ba44994f66d4/lm_eval/__main__.py#L65)) `--num_fewshot` to be an `int` and `--limit` to be a `float`.  it looks like the code here is passing in strings.  
There's some code in `lm-eval` that might be translating the types here...but I'm not sure.

Reply:
Im pretty sure this is the right way to parse arguments to a bash script. I followed the same structure we have across all files

Reply:
lm-eval translates the types